### ISSUE TITLE
multiple uploads

### ISSUE THREAD

**Issue by boyriot** [sachin36987/google_drive_remote-upload_php#1]
could you add more features to upload multiple files?

**Comment by sachin36987** [comment]
Yeah! I will change few things with this included but after 1st of June. Because I don't have time right now.
